platform_login/package.json

```diff
@@ -1,34 +1,34 @@
 {
-    "name": "auth",
-    "version": "0.0.1",
-    "description": "Authentication and Token Service",
-    "main": "handler.js",
-    "private": true,
-    "author": [],
-    "scripts": {
-        "test": "mocha"
-    },
-    "dependencies": {
-		"chai": "^3.5.0",
-		"mocha": "^v3.0.0",
-		"deasync":"*",
-		"async":"*",
-		"jsonwebtoken": "*",
-		"request":"*",
-		"aws-sdk":"*",
-		"moment":"*",
-		"amazon-cognito-identity-js": "^1.19.0"
-    },
-    "devDependencies": {
-                "activedirectory":"*",
-                "deasync":"*",
-                "async":"*",
-                "jsonwebtoken": "*",
-                "request":"*",
-                "aws-sdk":"*",
-                "moment":"*",
-                "amazon-cognito-identity-js": "^1.19.0",
-                "chai": "^3.5.0",
-                "mocha": "^v3.0.0"		
-    }
+  "name": "login",
+  "version": "0.0.1",
+  "description": "Authentication and Token Service",
+  "main": "index.js",
+  "private": true,
+  "author": [],
+  "scripts": {
+    "test": "mocha"
+  },
+  "dependencies": {
+    "chai": "^3.5.0",
+    "mocha": "^v3.0.0",
+    "deasync": "*",
+    "async": "*",
+    "jsonwebtoken": "*",
+    "request": "*",
+    "aws-sdk": "*",
+    "moment": "*",
+    "amazon-cognito-identity-js": "^1.19.0"
+  },
+  "devDependencies": {
+    "activedirectory": "*",

```

Comment:
Remove the ones that are in dependencies. I also think activedirectory isn't required here (haven't looked at the code)

Reply:
Done.